docs/src/main/asciidoc/security-architecture-concept.adoc

```diff
@@ -23,19 +40,26 @@ In other contexts, it is possible to have other parallel representations of the
 
 For more information, see xref:security-identity-providers-concept.adoc[Identity providers].
 
+=== `SecurityIdentityAugmentor`
+Because Quarkus Security is customizable, for example, you can add authorization roles to `SecurityIdentity`, you can register and prioritize one or more custom security augmentors.
+
+Registered instances of `SecurityIdentityAugmentor` are invoked during the final stage of the security authentication process.
+For more information see the xref:security-customization.adoc#security-identity-customization[Security Identity Customization] section of the "Security Tips and Tricks" guide.
```

Comment:
Re line 47:
Small typo: missing comma after "For more information....."

Reply:
Good catch. 

Thanks for reviewing Sheila. As this is merged, I'll submit a fix asap!